anesthetic/samples.py

```diff
@@ -630,6 +634,10 @@ def posterior_points(self, beta=1):
         """Get equally weighted posterior points at temperature beta."""
         return self.set_beta(beta).compress(-1)
 
+    def prior_points(self, params=None):
+        """Get prior points, i.e. posterior points at infinite temperature."""
```

Comment:
I think the docstring here should probably also mention equally weighted (and avoid i.e. as this is an implementation detail that will probably not be of use to users who only interact with the API), e.g. `"""Get equally weighted prior points."""`